### ISSUE TITLE
stitch order doesnt stitch

### ISSUE THREAD

**Issue by cmraaron** [inorbit-ai/ros_amr_interop#40]
https://github.com/inorbit-ai/ros_amr_interop/blob/d1971a43d457e12197166aaefd224a1dca979574/vda5050_connector/vda5050_connector_py/vda5050_controller.py#L1019

I think this needs to be `node_states` (no `s` on node)

Also https://github.com/inorbit-ai/ros_amr_interop/blob/d1971a43d457e12197166aaefd224a1dca979574/vda5050_connector/vda5050_connector_py/vda5050_controller.py#L1030-L1035

I think should take from `self._current_order` not `order` as they later have the order again appended to form the new order thusly:
https://github.com/inorbit-ai/ros_amr_interop/blob/d1971a43d457e12197166aaefd224a1dca979574/vda5050_connector/vda5050_connector_py/vda5050_controller.py#L1040-L1041

I may be able to contribute a PR if I can get some advice on how to run the tests, specifically just those in https://github.com/inorbit-ai/ros_amr_interop/blob/humble-devel/vda5050_connector/test/test_vda5050_controller.py, running `colcon test` just runs the linters?

**Comment by leandropineda** [comment]
Hi @cmraaron .

All tests should run with `colcon test`. The instructions on how to run the them can be found [here](https://github.com/inorbit-ai/ros_amr_interop/tree/humble-devel/vda5050_connector#testing).

```
colcon test --packages-select vda5050_connector && colcon test-result --verbose
```

In case you are not doing it, I'd recommend using the `development` docker container on the [examples repository](https://github.com/inorbit-ai/vda5050_adapter_examples/tree/galactic-devel/docker). Perhaps a missing configuration on your local environment is preventing tests from being discovered?

I'll take a thorough look at your findings next week and report back. Looking forward for your contributions!

**Comment by bobbleballs** [comment]
#42 Should fix this.

**Comment by cmraaron** [comment]
addressed by #42 many thanks @bobbleballs !
